diff -urN icecast-2.3.1.org/src/auth_url.c icecast-2.3.1/src/auth_url.c
--- icecast-2.3.1.org/src/auth_url.c	2005-11-30 19:16:17.000000000 +0100
+++ icecast-2.3.1/src/auth_url.c	2007-01-04 19:16:13.376846750 +0100
@@ -518,7 +518,9 @@
     curl_easy_setopt (url_info->handle, CURLOPT_WRITEDATA, url_info->handle);
     curl_easy_setopt (url_info->handle, CURLOPT_NOSIGNAL, 1L);
     curl_easy_setopt (url_info->handle, CURLOPT_TIMEOUT, 15L);
+#if CURLOPT_PASSWDFUNCTION != 0
     curl_easy_setopt (url_info->handle, CURLOPT_PASSWDFUNCTION, my_getpass);
+#endif
     curl_easy_setopt (url_info->handle, CURLOPT_ERRORBUFFER, &url_info->errormsg[0]);
 
     if (url_info->username && url_info->password)
